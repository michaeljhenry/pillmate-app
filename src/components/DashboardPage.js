import React, {Component} from 'react';
import {Link} from 'react-router-dom';
import Select from 'react-select';
import TimesList from '../components/TimesList'

const options = [
  { value: 'chocolate', label: 'Chocolate' },
  { value: 'strawberry', label: 'Strawberry' },
  { value: 'vanilla', label: 'Vanilla' }
]

const DashboardPage = () => {

  return(
    <div>
        Dashboard page content
        <Link className = 'button' to='/create'>Add Time</Link>
        <TimesList/>
    </div>
  )
};

export default DashboardPage;