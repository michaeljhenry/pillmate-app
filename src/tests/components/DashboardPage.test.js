import React from 'react'
import {shallow} from 'enzyme'
import DashboardPage from '../../components/DashboardPage';

test('tests rendering of dashboard page', () => {
    const wrapper = shallow(<DashboardPage/>);
    expect(wrapper).toMatchSnapshot();
});