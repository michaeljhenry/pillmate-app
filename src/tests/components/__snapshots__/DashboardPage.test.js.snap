// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests rendering of dashboard page 1`] = `
<div>
  Dashboard page content
</div>
`;
